.github/workflows/client_update.yml

```
@@ -0,0 +1,22 @@
+name: Trigger Client Update
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'server/api/tumdev/**'
```

I think limiting this to the file which is actually being used may be smart

```suggestion
      - 'server/api/tumdev/campus_backend.proto'
```

or to limit this to `proto` files
```suggestion
      - 'server/api/tumdev/*.proto'
```